server/models/index.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const process = require('process');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.json')[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
```

Comment:
are we going to need to set up `.env` files to run the server? If yes, please add a note about it in the PR.

Reply:
As of now no but this might be something we would want to do in the future.